tests/foreman/cli/test_ansible.py

```diff
@@ -103,3 +104,50 @@ def test_positive_ansible_e2e(target_sat, module_org, rhel_contenthost):
     assert SELECTED_ROLE, (
         SELECTED_VAR not in target_sat.cli.Ansible.variables_info({'name': SELECTED_VAR}).stdout
     )
+
+
+@pytest.mark.tier2
+def test_add_and_remove_ansible_role_hostgroup(target_sat):
```

Comment:
same scenario for adding and removing roles exists for hosts too, can you add a test to cover hosts part as well?

Reply:
Yeah we already have an e2e test case I will add an assertion there for `add` functionality
https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/cli/test_ansible.py#L27